backend/development.ini.sample

```diff
@@ -419,7 +419,7 @@ webdav.listen = localhost:3030
 ####
 
 ### Main Config for Caldav ##
-caldav.enabled = True
+caldav.enabled = False
```

Comment:
Why we change this default value?

Reply:
Deprecated with PR: https://github.com/tracim/tracim/pull/2618
It was not really clear if agenda feature should be activated by default as this need another process to be runned. But app refactor clarify behavior: agenda is default app and caldav.enabled parameter is deprecated.

Reply:
ok. So this parameter not more exist when refactor of app is merged in this branch.

Reply:
#2618 is merged. @inkhey you can update code and or resolve this conversation when this MR is rebased/fixed.

Reply:
this is done. viewed development.ini.sample here is outdated.